src/Og.php

```diff
@@ -426,8 +426,8 @@ protected static function getFieldBaseDefinition($plugin_id) {
    * @return OgSelection
    * @throws \Exception
    */
-  public static function getSelectionHandler($entity, $bundle, $field_name, array $options = []) {
-    $field_definition = FieldConfig::loadByName($entity, $bundle, $field_name);
+  public static function getSelectionHandler($entity_type_id, $bundle_id, $field_name, array $options = []) {
```

Comment:
In a follow up we should consider just passing on a field definition as a parameter here. This method is doing more than it should I think

```php
public static function getSelectionHandler(FieldDefinitionInterface $field_definition, array $options = []) {
```